libs/entry-components/styles/modules/components/buttons/_radio.scss

```diff
@@ -7,8 +7,12 @@ $radio-circles-selector: '.mdc-radio__outer-circle, .mdc-radio__inner-circle';
 	.mat-mdc-radio-group .mat-mdc-radio-button {
 		&.mat-mdc-radio-checked .mdc-radio {
 			&:not(:disabled):active #{$radio-selector}, #{$radio-selector} {
-				#{$radio-circles-selector} {
-					border-color: map.get($theme, 'general', 'buttons', 'radio', 'background');
+				$radio-border: map.get($theme, 'general', 'buttons', 'radio', 'background');
+
+				@if $radio-border and $radio-border != transparent {
```

Comment:
Why's transparent important in this case?

Reply:
I experimented with transparent values and found that the radio button becomes invisible. In other scenarios, we allow the default values to prevail, so it would be beneficial to address that aspect as well.

Reply:
mkay, but we haven't implemented odd/even coloring if transparent is passed (as described in main ticket) in entry-components\styles\modules\components\tables\_rows.scss

The idea is NOT to use primary color if transparent is given but to use underlying color (odd or even, depending on row itself)